Docker/development_docker-compose.yml

```diff
@@ -3,6 +3,9 @@ version: '2'
 services:
   db:
     image: postgres:9.5
+    environment:
```

Comment:
@Snehal1894 what is this for could comment briefly ?

Reply:
There are updates in Postgres 9.x images available on Docker Hub, for security purposes. Now we either need to set password for specific user to get the connect or we can explicity mention that we allow all connection. We have done the later.
Git hub link for this change is : https://github.com/docker-library/postgres/pull/658

Reply:
#73